module/validation/receipt_validator_test.go

```diff
@@ -50,7 +50,7 @@ func (s *ReceiptValidationSuite) TestReceiptValid() {
 		unittest.WithResult(valSubgrph.Result))
 	s.AddSubgraphFixtureToMempools(valSubgrph)
 
-	receiptID := receipt.ID()
+	receiptID := receipt.SignableID()
```

Comment:
```suggestion
	receiptSignableID := receipt.SignableID()
```
I think we should avoid naming instances of "signable ID" as just "id" or "receiptID", to avoid confusion with the canonical ID for the model.